### ISSUE TITLE
java: Add tags for native image testing docker image 

### ISSUE THREAD

**Issue by mpeddada1** [googleapis/testing-infra-docker#253]
Should we use a tag when specifying the container image rather than “latest”? With such set up, we can create one pull request that upgrades the version of both shared dependencies BOM and the container image together.

_Originally posted by @suztomo in https://github.com/googleapis/testing-infra-docker/issues/252#issuecomment-1286008661_

TODO:
In addition tagging the docker image, the kokoro configs for the presubmit jobs also need to be updated to take the tagged image into account: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/java_library/.kokoro/presubmit/graalvm-native.cfg
      
cc/ @suztomo

**Comment by suztomo** [comment]
When using, Kokoro configuration file can reference the tag like this: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.kokoro/appengine/node10/common.cfg#L21

```
env_vars: {
    key: "TRAMPOLINE_IMAGE"
    value: "gcr.io/cloud-devrel-kokoro-resources/node:10-user"
}
```

**Comment by suztomo** [comment]
Node's docker image https://github.com/googleapis/testing-infra-docker/blob/main/node/cloudbuild.yaml#L20

> Tag an image (-t)

https://docs.docker.com/engine/reference/commandline/build/#tag-an-image--t

**Comment by mpeddada1** [comment]
Closed via #254
